lib/cmds/login.ts

```diff
@@ -55,28 +54,25 @@ export const login = async ({
     token = managementTokenFlag
   } else {
     if (managementToken) {
-      log()
-      log(
-        `Looks like you already stored a management token on your system. ${chalk.dim(
-          `(Located at ${await getConfigPath()})`
-        )}`
+      console.log(
```

Comment:
Why did we drop the `log` utility?

Reply:
The `log` utility is just a wrapper on the `console.log`.
The others utility have a bit more of logic/styling involved, but we could keep it as log for consistency 🤔 
https://github.com/contentful/contentful-cli/blob/master/lib/utils/log.js

Reply:
Yup, it's just a wrapper that confuses me with other utility loggers we have, overtime I have to click to see what it is, I'd say it's not worth it. Maybe in another PR I'll just do a quick find and replace

Reply:
I think that's okay but let's kill it everywhere then, yeah.